cmd/oceanbench/broadcast.go

```diff
@@ -110,44 +110,47 @@ type Settings struct {
 
 // BroadcastConfig holds configuration data for a YouTube broadcast.
 type BroadcastConfig struct {
```

Comment:
Any reason why we cannot define BroadcastConfig in one place, and share it between Bench and TV?

Reply:
Can't really think of good reason. It has been annoying keeping them in sync.